src/com_tjvendors/admin/tables/vendor.php

```diff
@@ -219,6 +219,10 @@ public function check()
 				}
 
 				$filename   = JFile::stripExt($singleFile['name']);
+
+				// Strip unwanted characters
+				$filename   = strip_tags($filename);
```

Comment:
@Mayaank12 @snehalpatil0904 Can we use `JFile::makeSafe($filename);` instead?

Reply:
Updated